pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractReplicator.java

```diff
@@ -243,6 +250,11 @@ public synchronized CompletableFuture<Void> disconnect(boolean failIfHasBacklog)
         return closeProducerAsync();
     }
 
+    public CompletableFuture<Void> close() {
```

Comment:
Should we also modify the other places where disconnect() and disconnect(boolean) are being called?

Reply:
Here I will change the implementation method. The current implementation is not complete yet.